src/device-registry/bin/www

```diff
@@ -30,7 +30,7 @@ const eventSchema = Joi.object({
     "longitude": Joi.number().precision(5).optional(),
     "satellites": Joi.number().optional(),
     "hdop": Joi.number().optional(),
-    "altitude": Joi.number().optional(),
+    "altitude": Joi.number().integer().min(-Infinity).max(Infinity).optional(),
```

Comment:
During KAFKA consumption in device-registry, allow any integer value for the altitude field during input validation

Reply:
@BenjaminSsempala , let us follow up with either @deogratius2phantom or @semajoel about the valid ranges for our altitude sensors. cc: @NoahNsimbe